examples/server.rs

```diff
@@ -37,17 +36,20 @@ fn main() {
 }
 
 struct Svc;
-impl Service<Request<Body>> for Svc {
-    type Response = Response<LiftBody<Body>>;
+impl<B> Service<Request<B>> for Svc
+    where B: tower_http::Body
+{
+    type Response = Response<Vec<u8>>;
     type Error = hyper::Error;
     type Future = future::FutureResult<Self::Response, Self::Error>;
 
     fn poll_ready(&mut self) -> Poll<(), Self::Error> {
         Ok(().into())
     }
 
-    fn call(&mut self, _req: Request<Body>) -> Self::Future {
-        let body = LiftBody::new(Body::from("Hello World!"));
+    fn call(&mut self, _req: Request<B>) -> Self::Future {
+        // let body = LiftBody::new(Body::from("Hello World!"));
+        let body = b"Hello World!".to_vec();
```

Comment:
minor thing might be nice to show this as `req.into_body()` then pass that back.